lib/fishjam_web/telemetry.ex

```diff
@@ -108,6 +108,25 @@ defmodule FishjamWeb.Telemetry do
         #   - WebSocket traffic
         #   - RTP events (RTSP components don't use ICE)
         #   - HTTP traffic related to metrics (not handled by Phoenix)
+
+        counter("fishjam.rpc_client.call.count",
+          event_name: [:fishjam, :rpc_client, :call],
+          measurement: :duration
+        ),
+        counter("fishjam.rpc_client.multicall.count",
+          event_name: [:fishjam, :rpc_client, :multicall],
+          measurement: :duration
+        ),
+        metric_type.("fishjam.rpc_client.call.duration.milliseconds",
```

Comment:
Yeah, that's pretty much what has to be done, unfortunately.

`last_value` isn't really very useful to us, it gives us only one reading at a time (the duration of the last recorded request/RPC in this case), which, since we scrape the metrics from the Prometheus endpoint every 10 seconds, isn't really representative. [`summary` performs some aggregations](https://prometheus.io/docs/practices/histograms/) and is better suited here, but the library doesn't support it

Reply:
I'm not sure, but based [on this line](https://github.com/fishjam-dev/fishjam/blob/b264fed329ffc9d233a07df60a046ec6f6c86c95/lib/fishjam_web/telemetry.ex#L27) I think that they are presented with `metric_type` `last_value`.